cmd/dbseed/exchange.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	exchangeDB "github.com/thrasher-corp/gocryptotrader/database/repository/exchange"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/urfave/cli/v2"
+)
+
+var seedExchangeCommand = &cli.Command{
+	Name:  "exchange",
+	Usage: "seed exchange data",
+	Subcommands: []*cli.Command{
+		{
+			Name:      "file",
+			Usage:     "seed exchange data from a file",
+			ArgsUsage: "<flags>",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:  "filename",
+					Usage: "<filename>",
+				},
+			},
```

Comment:
This does not read from the same path (curr dir) as with `.\dbseed.exe candle file --filename=hi.csv`
```
2020/07/28 10:59:00 Stat : The system cannot find the path specified.
```

This also requires documenting what's required in the file

Reply:
TakesFile & FilePath params have been added with FilePath defaulting to os.WorkDir() or "." if that returns an error this now allows both relative paths and searching of current dir